pilot/pkg/bootstrap/server.go

```diff
@@ -76,6 +81,14 @@ var (
 		plugin.Health,
 		plugin.Mixer,
 	}
+
+	IstiodAddress = env.RegisterStringVar("ISTIOD_ADDR", "",
+		"Local service address of istiod. Default value is mesh.defaultConfig.discoveryAddress, use this to override if the discovery address is external")
+
+	// IstiodNamespace defines the namespace where istiod runs. It is based on the POD_NAMESPACE, generated by K8S env,
```

Comment:
This is not istiod

Reply:
It kind of is... We are merging istiod functionality into pilot so we can accelerate testing - but eventually the service name and binary will be called istiod once we remove the other components. AFAIK the agreement was to name this combined control plane 'istiod', even if it 
is based on pilot.